static/js/protein/chart.ts

```diff
@@ -17,13 +17,36 @@
 import * as d3 from "d3";
 import _ from "lodash";
 
+//import styles from './protein.scss';
 import { InteractingProteinType } from "./page";
 import { ProteinVarType } from "./page";
 const SVGNS = "http://www.w3.org/2000/svg";
 const XLINKNS = "http://www.w3.org/1999/xlink";
 const HEIGHT = 224;
 const WIDTH = 500;
-const MARGIN = { top: 30, right: 30, bottom: 70, left: 90 };
+const MARGIN = { top: 30, right: 30, bottom: 90, left: 160 };
+// bar chart color for most of the charts
+const BAR_COL = "maroon";
+// tooltip style
+const TOOL_TIP = d3
+  .select("div")
```

Comment:
would you recommend changing tooltip from a constant to a variable since each chart will its own specific id, instead of the generic div ?

Reply:
I think could add a function `addTooltip(containerId: string)` that creates the tooltip element and returns it. Then for each chart, can call this function and save the tooltip into a variable so it can be modified in the mouse events.